Dockerfile.test

```diff
@@ -0,0 +1,17 @@
+FROM registry.access.redhat.com/ubi8/python-36:latest
+
+USER root
+
+WORKDIR /managedtenant-cli
+
+COPY . ./
+
+RUN pip install pipenv
+
+RUN pipenv lock --pre --clear
+
+RUN pipenv install --dev
```

Comment:
This has been a source of pain and misery for me in the past. It hangs and fails often, so much that I would instead do:

```bash
$ pipenv lock -r > requirements.txt
$ pip install -r requirements.txt
```